odin recipe project

Sushilkumar3255 opened the pull request:
Odin Recipes Project

Description:
In this PR a page is designed such that it contain multiple links to their specific pages with the use of anchor tag.
I have done this with basic concepts of HTML in this project.

Sushilkumar3255 committed:
odin recipe project

roopaljasnani commented on index.html:
```diff
@@ -0,0 +1,17 @@
+        <li><a href="./recipes/Dal-Makhani.html">Dal Makhani Recipe</a></li>
+    </ol>
+</body>
+</html>
```
make sure to have a blank new line at the end of each file.

Use `Prettier` plugin (available in VSCode extensions) and configure the settings to auto format your files on Save

roopaljasnani reviewed (commented):
While creating PRs that involve some front-end/design work, make sure to attach screenshots/screen recordings in the PR description. This serves as a valid Test Plan & allows the PR reviewer to see what your changes look like and does not require them to check out your branch.

Also, it is important that we fill in available metadata information like `Assignees`, `Reviewers`,

Sushilkumar3255 committed:
On branch Recipes
 Your branch is up to date with 'origin/Recipes'.

Description:
	added line at end of each files.

 Changes to be committed:
	modified:   index.html
	modified:   recipes/Dal-Makhani.html
	modified:   recipes/Pizza.html
	modified:   recipes/veg-burger.html

Sushilkumar3255 commented:
![recipe01](https://user-images.githubusercontent.com/83236394/233336893-c17661c4-825e-48a4-abd7-173f3658b3e2.png)
![recipe02](https://user-images.githubusercontent.com/83236394/233336925-d66f7e80-b7b7-4463-bdb6-25fff3983eac.png)
![recipe03](https://user-images.githubusercontent.com/83236394/233336978-b83cb90b-3c9c-49ec-9f2a-512a232080c6.png)
